src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -81,7 +81,8 @@ public Shooter(Vision vision) {
         SmartDashboard.putNumber("Target Hood Angle", 60);
         SmartDashboard.putNumber("TSR", -5500);
 
-        try (BufferedReader br = new BufferedReader(new FileReader("/home/lvuser/ShooterData.csv"))) {
+        try {
```

Comment:
I think because you moved the BufferedReader out of the try() clause, you need to explicitly close it. However, I believe the previous format is better, because it will auto close it even with an exception.

Reply:
ok, I'll change the beginning syntax back, did I get the actual error right?

Reply:
and should I clean out the unused imports?

Reply:
Yes, you found the bug!! And yes, please try to clean out an of the obvious warning (eg the import ones)